### DISCUSSION TITLE
Settings bugs/improvements

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#2003] [Ideas] [answered]
1. 'Live burst' option is greyed out and cannot be toggled
<img width="1000" alt="Screenshot 2021-10-14 at 16 06 18" src="https://user-images.githubusercontent.com/30378821/137345539-cdff9aa8-c147-499f-bca8-a99b86e60ecf.png">

2. 'Helvetica' font is not Helvetica
3. Font size is in arbitrary units instead of point size
<img width="965" alt="Screenshot 2021-10-14 at 16 06 36" src="https://user-images.githubusercontent.com/30378821/137345683-27840142-46fe-4ca3-a411-592325618dd6.png">

**Comment by Miodec** [comment, marked as answer]
1. Ill fix that
2. Helvetica being Comic Sans is a joke.
3. For font size, think of it as a multiplier ( 1x standard, 3x standard and so on, the standard being the size that's defined in your browser settings)

**Reply by unrealapex** [reply]
Maybe have the rendered font size appear in a tooltip supplementing the rem size?

**Comment by cryancampbell** [comment]
question - what percent of users have "Helvetica" set as their font (as in - are looking at comic sans but under the helvetica setting)? assuming you can answer of course

i cackled when i saw it load CS, great joke @Miodec

**Reply by Miodec** [reply]
;)

EDIT: To actually answer the question - I dont have a statistic on how many people actually clicked the Helvetica button sadly.

**Reply by unrealapex** [reply]
Probably a lot of people lol
